scripts/setup-gitlab.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+GITLAB_WEBSERVICE_POD=$(kubectl get pods --namespace gitlab -l app=webservice -o name | sed 's|pod/||')
+
+while true; do
+  # Get the pod status using kubectl
+  POD_STATUS=$(kubectl get pods -n gitlab -l app=webservice -o jsonpath='{.items[0].status.phase}')
+
+  # Check if the pod is in 'Running' state
+  if [ "${POD_STATUS}" = "Running" ]; then
+    echo "Pod is ready, continuing with next steps..."
+    sleep 10
+    break  # Exit the loop
+  else
+    echo "Pod is not ready yet. Current status: $POD_STATUS"
+    sleep 5  # Wait for 5 seconds before checking again
+  fi
+done
+
+GITLAB_TOOLBOX_POD=$(kubectl get pods --namespace gitlab -l app=toolbox -o name | sed 's|pod/||')
+kubectl cp ./scripts/gitlab-setup.rb gitlab/${GITLAB_TOOLBOX_POD}:/tmp 
```

Comment:
Lets apply it as config map and then mount is as volume using the helm chart.
Some example: https://gitlab.com/gitlab-org/gitlab/-/issues/375260